Gem/Code/Source/MultiplayerSampleAWSGameLiftServerSystemComponent.cpp

```diff
@@ -72,7 +76,31 @@ namespace MultiplayerSample
     bool MultiplayerSampleAWSGameLiftServerSystemComponent::OnCreateSessionBegin([[maybe_unused]] const Multiplayer::SessionConfig& sessionConfig)
     {
         // Add here: additional logic, if needed, to ready server process for hosting a session.
-        // For now, sufficient to return true so Amazon GameLift knows server process is responsive.
+
+        AzFramework::LevelLoadBlockerBus::Handler::BusDisconnect();
+
+        const char* level = m_loadedLevelName ? m_loadedLevelName : "Levels/NewStarbase/NewStarbase.spawnable";
```

Comment:
could/should we make this default level value a .setreg entry or cvar that is set in a .setreg?

Reply:
Actually, if m_loadedLevelName is empty, shouldn't it just not load anything, and then just not block the next LoadLevel command that comes through? It seems like the concept here is that if the level load happens first, the name gets saved so we can defer and load it later, but if the level load happens second, it should just happen?

Reply:
I don't think anything will be calling `LoadLevel` after this, though, and we really can't wait to do so if this function is being called, b/c it means Amazon GameLift wants MPS to start hosting players. So it's less "don't load the level too soon" and more: "only load the level at this exact time".

We could actually skip the `ShouldBlockLevelLoading` call's attempt to store the tried-to-load level altogether, and just always pull from a CVAR or setreg (or hardcoded value!) which level we want to load, but including it friendlier to use with our current conventions of running & executing the launchers with .cfg files or command line args.

Reply:
I'm 100% cool with not loading NewStarbase as a fallback if the user never asks to load a level. Really I see this change as a way to keep user expectations. IE: If I call loadlevel, a level _SHOULD_ be loaded. Without this change I could call loadlevel, but no level gets loaded if GameLift is still initializing. That being said, this is game code, so in reality, will anyone be connecting to gamelift and not want to load newstarbase?

_"I don't think anything will be calling LoadLevel after this"_
This is left up to the user. It's entirely possible the developer launchers their server, connects to gamelift, but then wants to load some other level.

Reply:
> it's just the hard-coded name that i'm not fond of. but if that were put in a cvar or setreg, then we're now starting to list the same level name in multiple setreg keys and/or cvars to get it to load, which seems pretty messy.

Ah! Ok yes, agreed, introducing a hierarchy of "pull the value from here, then from here", etc., would get messy. If we want to just enforce the existing expectation that you run this with a .cfg file or command line arg that specifies the level (which we catch and load at the right time), and forgo any kind fallback (hardcoded or otherwise), I am totally cool with that! Will remove the fallback.